x-pack/legacy/plugins/ml/public/data_visualizer/components/field_data_card/metric_distribution_chart/metric_distribution_chart.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  AreaSeries,
+  Axis,
+  Chart,
+  CurveType,
+  DataSeriesColorsValues,
+  getAxisId,
+  getSpecId,
+  Position,
+  ScaleType,
+  Settings,
+  TooltipValueFormatter,
+  TooltipValue,
+} from '@elastic/charts';
+
+import chrome from 'ui/chrome';
```

Comment:
Should be able to use `uiContext` as of https://github.com/elastic/kibana/pull/42244

Reply:
Replaced this and the two other uses of `ui/chrome` in https://github.com/elastic/kibana/pull/42685/commits/31da8259e659eb908cba4922d4ab44bd0dd3898c